main.go

```diff
@@ -31,22 +31,27 @@ const (
 	diskPathConverted    string = "./tmp/disk.qcow2"
 )
 
-func applyVirtualMachineExport(vmName string) error {
+func applyVirtualMachineExport(vmNamespace, vmName string) error {
 	log.Println("Applying VirtualMachineExport object...")
 
 	client, err := kubecli.GetKubevirtClient()
 	if err != nil {
 		return err
 	}
 
-	vmNamespace := os.Getenv("VM_NAMESPACE")
+	env := os.Getenv("VM_NAMESPACE")
```

Comment:
Please add to documentation, that `VM_NAMESPACE` env variable is used if both `VM_NAMESPACE` env variable and `vmnamespace` param are set. Because it can happen that both will be populated and then user might be confused why it used different namespace

Reply:
Sure! I've added it.